Add API docs page

hmasila opened the pull request:
## Description
This adds the API docs page for the evil insult generator API.

## Screenshops
<img width="1305" alt="screen shot 2018-10-25 at 10 21 14" src="https://user-images.githubusercontent.com/20906768/47544478-e7a60280-d8ef-11e8-830e-d7e4e483209f.png">

EvilInsultGenerator commented:
Thank you so much for your PR, @hmasila! It is really helpful!

Could you please rename the file from `evil_insults.html` to `index.html`?

EvilInsultGenerator commented:
By the way, can you please add a note mentioning the [fallback server](https://slave.evilinsult.com/generate_insult.php?lang=en&type=json)?
And is it possible to make the page mobile-friendly?

hmasila commented:
I have updated the name to `index.html` and also added the fallback server. About the page being mobile friendly, I think it is responsive enough
<img width="392" alt="screen shot 2018-11-03 at 12 35 58" src="https://user-images.githubusercontent.com/20906768/47950582-58d75c80-df65-11e8-9c55-0d5abd4e2f2b.png">
